### ISSUE TITLE
Emoji

### ISSUE THREAD

**Issue by Hoodgail** [airmessage/airmessage-web#18]
![image (6)](https://user-images.githubusercontent.com/45743294/177019916-82a427c6-bb4a-44e0-9fbf-c31efe82b3fc.png)

Button to trigger a components full of reactions. I'm getting a little bored of using Google to look up "laughing emoji" to send to friends. 🤣 🤣

**Comment by tagavari** [comment]
Thanks for the suggestion! I'll see if I can get around to this at some point

**Comment by Hoodgail** [comment]
> Thanks for the suggestion! I'll see if I can get around to this at some point

Np, you wounlnt need to make the component your self. Since there are many open source react versions, such as `emoji-mart`

**Comment by Geczy** [comment]
emoji mart is good but still doesn't support typescript, so it will not integrate with this project well

the only other option is emoji-picker-react but that one is extremely slow and bad for the site, lags everything

**Comment by tagavari** [comment]
@Geczy I see that you've created a PR with emoji-picker-react, thank you for that!

For reference, most popular NPM packages have types created for them with the `@types/` prefix. `emoji-mart` has [`@types/emoji-mart`](https://www.npmjs.com/package/@types/emoji-mart) on NPM. I haven't tested this package though.

If `emoji-picker-react` doesn't perform well, then it may be worth trying with `emoji-mart` again

**Comment by Geczy** [comment]
yeah even with @types/emoji-mart it was still failing in several places. that type is for v3 of emoji-mart, but the latest is v5 so maybe thats the issue :( there's an open issue on emoji-mart repo to update it to v5 :/

**Comment by Geczy** [comment]
i think its better to just use the system builtin emoji picker. no web emojipicker will ever top that

To open the emoji keyboard, hold the Windows key down, and press either the period . or semicolon ; key.

`Windows key + .`

Alternatively,

`Windows key + ;`

This keyboard shortcut will open an emoji keyboard like the one shown below. 

<img width="245" alt="image" src="https://user-images.githubusercontent.com/1036968/179540069-a550c7c8-f84f-47b5-b091-08d412af7ffc.png">

**Comment by Hoodgail** [comment]
@Geczy Not everyone uses windows. Does linux & macos have a native emoji picker that works like that?

**Comment by Geczy** [comment]
> @Geczy Not everyone uses windows. Does linux & macos & chrome OS have a native emoji picker that works like that?

yes
